test/e2e/pivoting_test.go

```diff
@@ -282,3 +291,105 @@ func labelBMOCRDs(targetCluster framework.ClusterProxy) {
 		Expect(err).To(BeNil(), "Cannot label BMO CRDs")
 	}
 }
+
+func rePivoting() {
```

Comment:
Let's add a log for beginning of re-pivoting tests here as well?

Reply:
fixed